Kiosk/App/CardHandler.swift

```diff
@@ -1,9 +1,14 @@
 import UIKit
-import ReactiveCocoa
+import RxSwift
 
 class CardHandler: NSObject, CFTReaderDelegate {
 
-    let cardSwipedSignal = RACSubject()
+    private let _cardStatus = BehaviorSubject(value: "")
```

Comment:
Is there any reason to have a `BehaviorSubject` instead of a `PublishSubject`? 
Later in this same file the `cardStatus` is then defined as an observable with the first value skipped, so what is the point to have an initial value if this is later skipped? I am asking b/c maybe there's a technical reason for this and I am curious :)